### PR TITLE
journal: implement GetUsage() (sd_journal_get_usage())

### PR DESCRIPTION
**Pull request by audebert**

### CODE REVIEW

File: sdjournal/journal.go [hunk 1]
```diff
@@ -237,3 +237,17 @@ func (j *Journal) Wait(timeout time.Duration) int {
 
 	return int(r)
 }
+
+// Returns the journal disk space usage, in bytes.
```
Comment (vcaputo) [inline sdjournal/journal.go]: Please update the docstring to be consistent with the rest:
GetUsage returns the journal disk space usage, in bytes.

File: sdjournal/journal_test.go [hunk 2]
```diff
@@ -68,3 +68,27 @@ func TestJournalFollow(t *testing.T) {
...
+	defer j.Close()
+
+	s, err := j.GetUsage()
+
+	if err != nil {
+		t.Fatalf("Error getting journal size: %s", err)
+	}
+
+	if s == 0 {
+		t.Fatalf("Journal size is 0.")
```
Comment (vcaputo) [inline sdjournal/journal_test.go]: It seems odd to fatal on 0 when the sd_journal_get_usage() manual says nothing about 0 size being invalid

**Comment by audebert** [comment]
Thank you for the review, I have updated the patch.

**Comment by vcaputo** [comment]
lgtm, thanks for the contribution!  We'll have to see about improving the tests overall in this package at some point.

**Pull request merged**